build.fsx

```diff
@@ -515,6 +515,10 @@ let testRust testMode =
     cleanDirs [buildDir </> "tests"]
     cleanDirs [buildDir </> ".fable"]
 
+    // copy rust only tests files (these must be present when running dotnet test as import expr tests for file presence)
+    makeDirRecursive (buildDir </> "tests" </> "src")
+    copyFile (projectDir </> "tests" </> "src" </> "ExtInteropTests.rs") (buildDir </> "tests" </> "src" </> "ExtInteropTests.rs")
```

Comment:
Perhaps copy all `*.rs` tests, so you don't have to change it every time.

Reply:
This would be far better. Can I just use a wildcard here?

Reply:
This doesn't seem to work.
```
    copyFile (projectDir </> "tests" </> "src" </> "ExtInteropTests.rs") (buildDir </> "tests" </> "src" </> "*.rs")
```

Reply:
fixed